### ISSUE TITLE
Stdout in SystemV init script

### ISSUE THREAD

**Issue by theossystems** [sbt/sbt-native-packager#722]
Now we don't save stdout/stderr of application in `SystemV` init system in `Debian` (but save in `Rpm`). It could be useful for example when app is crashed due to OOM exception. It this case nothing is logged and app is stopped.
I have a workaround that works for `start-stop-daemon` version >= 1.16.5 (Debian Wheezy has 1.16.16).
It could broke some installations if application wrote a lot to stdout and daemon.log will grow rapidly.
WDYT?

**Comment by muuki88** [comment]
I had trouble with `SystemV` as well. Actually I would like to keep things as stable as they are and hope that `SystemD` will be used more and more as a default systemloader, which brings all this features out-of-the-box with no error-prone bash needed.

If you provide a pull request that makes this feature selectable (e.g. a `systemVLog` `setting`), which is by default off and doesn't change current behaviour, we could merge it.

In a nutshell: `SystemV` is so old, error prone and hard to use and [SystemD](https://wiki.debian.org/systemd) is on it's way (Debian, Ubuntu, Fedora IIRC already adapted it). So if a change you have to make a pull request :)
